Index: tar-1.26-r1.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/app-arch/tar/tar-1.26-r1.ebuild,v
retrieving revision 1.8
diff -u -B -r1.8 tar-1.26-r1.ebuild
--- tar-1.26-r1.ebuild	20 Feb 2013 22:54:26 -0000	1.8
+++ tar-1.26-r1.ebuild	30 Mar 2013 11:08:29 -0000
@@ -4,7 +4,7 @@
 
 EAPI="3"
 
-inherit flag-o-matic eutils autotools
+inherit flag-o-matic eutils autotools usr-gentoo
 
 DESCRIPTION="Use this to make tarballs :)"
 HOMEPAGE="http://www.gnu.org/software/tar/"
@@ -38,7 +38,6 @@
 	FORCE_UNSAFE_CONFIGURE=1 \
 	econf \
 		--enable-backup-scripts \
-		--bindir="${EPREFIX}"/bin \
 		--libexecdir="${EPREFIX}"/usr/sbin \
 		$(usex userland_GNU "" "--program-prefix=g") \
 		$(use_enable nls) \
@@ -60,7 +59,7 @@
 	# the host system (FreeBSD, Solaris, Darwin) will be found instead
 	# of the Prefix provided (GNU) tar
 	if use prefix ; then
-		dosym tar /bin/gtar
+		dosym tar /usr/bin/gtar
 	fi
 
 	dodoc AUTHORS ChangeLog* NEWS README* THANKS
@@ -73,4 +72,6 @@
 			-type f -a '!' '(' -name tar -o -name ${p}tar ')' \
 			-delete
 	fi
+
+	usr_wrap_bin tar $(use prefix && echo gtar)
 }
