Index: pam-1.1.6-r4.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/sys-libs/pam/pam-1.1.6-r4.ebuild,v
retrieving revision 1.1
diff -u -B -r1.1 pam-1.1.6-r4.ebuild
--- pam-1.1.6-r4.ebuild	28 Feb 2013 05:59:27 -0000	1.1
+++ pam-1.1.6-r4.ebuild	31 Mar 2013 21:24:40 -0000
@@ -4,7 +4,7 @@
 
 EAPI=5
 
-inherit libtool multilib eutils pam toolchain-funcs flag-o-matic db-use autotools autotools-utils
+inherit libtool multilib eutils pam toolchain-funcs flag-o-matic db-use autotools autotools-utils usr-gentoo
 
 MY_PN="Linux-PAM"
 MY_P="${MY_PN}-${PV}"
@@ -101,9 +101,7 @@
 
 	local myeconfargs=(
 		--htmldir="${EPREFIX}"/usr/share/doc/${PF}/html
-		--libdir="${EPREFIX}"/usr/$(get_libdir) \
-		--enable-securedir="${EPREFIX}"/$(get_libdir)/security
-		--enable-isadir="${EPREFIX}"/$(get_libdir)/security
+		--enable-isadir=.
 		$(use_enable nls)
 		$(use_enable selinux)
 		$(use_enable cracklib)
@@ -131,15 +129,10 @@
 src_install() {
 	autotools-utils_src_install sepermitlockdir="${EPREFIX}/run/sepermit"
 
-	# Need to be suid
-	fperms u+s /sbin/unix_chkpwd
-
-	gen_usr_ldscript -a pam pamc pam_misc
-
 	# create extra symlinks just in case something depends on them...
 	local lib
 	for lib in pam pamc pam_misc; do
-		if ! [[ -f "${ED}"/$(get_libdir)/lib${lib}$(get_libname) ]]; then
+		if ! [[ -f "${ED}"/usr/$(get_libdir)/lib${lib}$(get_libname) ]]; then
 			dosym lib${lib}$(get_libname 0) /$(get_libdir)/lib${lib}$(get_libname)
 		fi
 	done
@@ -157,6 +150,11 @@
 d /run/sepermit 0755 root root
 EOF
 	fi
+
+	usr_move_all
+
+	# Need to be suid
+	fperms u+s /usr/sbin/unix_chkpwd
 }
 
 pkg_preinst() {
