Index: attr-2.4.46-r2.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/sys-apps/attr/attr-2.4.46-r2.ebuild,v
retrieving revision 1.2
diff -u -B -d -u -p -r1.2 attr-2.4.46-r2.ebuild
--- attr-2.4.46-r2.ebuild	21 Mar 2013 15:11:57 -0000	1.2
+++ attr-2.4.46-r2.ebuild	30 Mar 2013 21:49:47 -0000
@@ -4,7 +4,7 @@
 
 EAPI="4"
 
-inherit eutils toolchain-funcs
+inherit eutils toolchain-funcs usr-gentoo
 
 DESCRIPTION="Extended attributes tools"
 HOMEPAGE="http://savannah.nongnu.org/projects/attr"
@@ -39,8 +39,7 @@ src_configure() {
 	econf \
 		$(use_enable nls gettext) \
 		--enable-shared $(use_enable static-libs static) \
-		--libexecdir="${EPREFIX}"/usr/$(get_libdir) \
-		--bindir="${EPREFIX}"/bin
+		--libexecdir="${EPREFIX}"/usr/$(get_libdir)
 }
 
 src_install() {
@@ -49,6 +48,5 @@ src_install() {
 	# the man-pages packages provides the man2 files
 	rm -r "${ED}"/usr/share/man/man2
 
-	# we install attr into /bin, so we need the shared lib with it
-	gen_usr_ldscript -a attr
+	usr_wrap_bin {get,set}fattr attr
 }
