Index: coreutils-8.21.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/sys-apps/coreutils/coreutils-8.21.ebuild,v
retrieving revision 1.2
diff -u -B -d -u -p -r1.2 coreutils-8.21.ebuild
--- coreutils-8.21.ebuild	20 Feb 2013 22:37:32 -0000	1.2
+++ coreutils-8.21.ebuild	30 Mar 2013 22:10:43 -0000
@@ -4,7 +4,7 @@
 
 EAPI="3"
 
-inherit eutils flag-o-matic toolchain-funcs
+inherit eutils flag-o-matic toolchain-funcs usr-gentoo
 
 PATCH_VER="1.0"
 DESCRIPTION="Standard GNU file utilities (chmod, cp, dd, dir, ls...), text utilities (sort, tr, head, wc..), and shell utilities (whoami, who,...)"
@@ -132,21 +132,14 @@ src_install() {
 	newins src/dircolors.hin DIR_COLORS || die
 
 	if [[ ${USERLAND} == "GNU" ]] ; then
-		cd "${ED}"/usr/bin
-		dodir /bin
 		# move critical binaries into /bin (required by FHS)
 		local fhs="cat chgrp chmod chown cp date dd df echo false ln ls
 		           mkdir mknod mv pwd rm rmdir stty sync true uname"
-		mv ${fhs} ../../bin/ || die "could not move fhs bins"
 		# move critical binaries into /bin (common scripts)
 		local com="basename chroot cut dir dirname du env expr head mkfifo
 		           mktemp readlink seq sleep sort tail touch tr tty vdir wc yes"
-		mv ${com} ../../bin/ || die "could not move common bins"
 		# create a symlink for uname in /usr/bin/ since autotools require it
-		local x
-		for x in ${com} uname ; do
-			dosym /bin/${x} /usr/bin/${x} || die
-		done
+		usr_wrap_bin ${fhs} ${com} uname
 	else
 		# For now, drop the man pages, collides with the ones of the system.
 		rm -rf "${ED}"/usr/share/man
