Index: iproute2-3.8.0.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/sys-apps/iproute2/iproute2-3.8.0.ebuild,v
retrieving revision 1.1
diff -u -B -r1.1 iproute2-3.8.0.ebuild
--- iproute2-3.8.0.ebuild	28 Feb 2013 19:19:37 -0000	1.1
+++ iproute2-3.8.0.ebuild	30 Mar 2013 13:21:54 -0000
@@ -4,7 +4,7 @@
 
 EAPI="4"
 
-inherit eutils toolchain-funcs flag-o-matic multilib
+inherit eutils toolchain-funcs flag-o-matic multilib usr-gentoo
 
 if [[ ${PV} == "9999" ]] ; then
 	EGIT_REPO_URI="git://git.kernel.org/pub/scm/linux/kernel/git/shemminger/iproute2.git"
@@ -78,23 +78,23 @@
 
 src_install() {
 	if use minimal ; then
-		into /
 		dosbin tc/tc
+		usr_wrap_sbin tc
 		return 0
 	fi
 
 	emake \
 		DESTDIR="${D}" \
-		LIBDIR="${EPREFIX}"/$(get_libdir) \
-		SBINDIR="${EPREFIX}"/sbin \
+		LIBDIR="${EPREFIX}"/usr/$(get_libdir) \
+		SBINDIR="${EPREFIX}"/usr/sbin \
 		CONFDIR="${EPREFIX}"/etc/iproute2 \
 		DOCDIR="${EPREFIX}"/usr/share/doc/${PF} \
 		MANDIR="${EPREFIX}"/usr/share/man \
 		ARPDDIR="${EPREFIX}"/var/lib/arpd \
 		install
 
-	dodir /bin
-	mv "${ED}"/{s,}bin/ip || die #330115
+	dodir /usr/bin
+	mv "${ED}"/usr/{s,}bin/ip || die #330115
 
 	dolib.a lib/libnetlink.a
 	insinto /usr/include
@@ -103,7 +103,10 @@
 	if use berkdb ; then
 		dodir /var/lib/arpd
 		# bug 47482, arpd doesn't need to be in /sbin
-		dodir /usr/bin
-		mv "${ED}"/sbin/arpd "${ED}"/usr/bin/ || die
+		mv "${ED}"/usr/sbin/arpd "${ED}"/usr/bin/ || die
 	fi
+
+	cd "${ED}"/usr/sbin || die
+	usr_wrap_sbin *
+	usr_wrap_bin ip
 }
