Index: kmod-13.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/sys-apps/kmod/kmod-13.ebuild,v
retrieving revision 1.2
diff -u -B -r1.2 kmod-13.ebuild
--- kmod-13.ebuild	10 Apr 2013 03:12:15 -0000	1.2
+++ kmod-13.ebuild	14 Apr 2013 07:31:08 -0000
@@ -6,7 +6,7 @@
 
 VIRTUAL_MODUTILS=1
 
-inherit autotools eutils libtool multilib linux-mod
+inherit autotools eutils libtool multilib linux-mod usr-gentoo
 
 if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="git://git.kernel.org/pub/scm/utils/kernel/${PN}/${PN}.git"
@@ -62,8 +62,6 @@
 src_configure()
 {
 	econf \
-		--bindir=/bin \
-		--with-rootlibdir=/$(get_libdir) \
 		$(use_enable static-libs static) \
 		$(use_enable tools) \
 		$(use_enable debug) \
@@ -80,15 +78,17 @@
 	if use tools; then
 		local bincmd sbincmd
 		for sbincmd in depmod insmod lsmod modinfo modprobe rmmod; do
-			dosym /bin/kmod /sbin/${sbincmd}
+			dosym ../bin/kmod /usr/sbin/${sbincmd}
 		done
 
 		# These are also usable as normal user
 		for bincmd in lsmod modinfo; do
-			dosym kmod /bin/${bincmd}
+			dosym kmod /usr/bin/${bincmd}
 		done
 	fi
 
+	usr_wrap_all
+
 	cat <<-EOF > "${T}"/usb-load-ehci-first.conf
 	softdep uhci_hcd pre: ehci_hcd
 	softdep ohci_hcd pre: ehci_hcd
