Index: systemd-201.ebuild
===================================================================
RCS file: /var/cvsroot/gentoo-x86/sys-apps/systemd/systemd-201.ebuild,v
retrieving revision 1.16
diff -u -B -r1.16 systemd-201.ebuild
--- systemd-201.ebuild	19 Apr 2013 00:25:16 -0000	1.16
+++ systemd-201.ebuild	20 Apr 2013 17:26:21 -0000
@@ -5,7 +5,7 @@
 EAPI=5
 
 PYTHON_COMPAT=( python2_7 )
-inherit autotools-utils linux-info multilib pam python-single-r1 systemd toolchain-funcs udev user
+inherit autotools-utils linux-info multilib python-single-r1 systemd toolchain-funcs udev user usr-gentoo
 
 DESCRIPTION="System and service manager for Linux"
 HOMEPAGE="http://www.freedesktop.org/wiki/Software/systemd"
@@ -103,7 +103,6 @@
 src_configure() {
 	local myeconfargs=(
 		--localstatedir=/var
-		--with-pamlibdir=$(getpam_mod_dir)
 		# make sure we get /bin:/sbin in $PATH
 		--enable-split-usr
 		# disable sysv compatibility
@@ -141,9 +140,6 @@
 		QUOTACHECK=/usr/sbin/quotacheck
 	)
 
-	# Keep using the one where the rules were installed.
-	MY_UDEVDIR=$(get_udevdir)
-
 	if use firmware-loader; then
 		myeconfargs+=(
 			--with-firmware-path="/lib/firmware/updates:/lib/firmware"
@@ -156,23 +152,10 @@
 	autotools-utils_src_configure
 }
 
-src_compile() {
-	autotools-utils_src_compile \
-		udevlibexecdir="${MY_UDEVDIR}"
-}
-
 src_install() {
 	autotools-utils_src_install -j1 \
-		udevlibexecdir="${MY_UDEVDIR}" \
 		dist_udevhwdb_DATA=
 
-	# keep udev working without initramfs, for openrc compat
-	dodir /bin /sbin
-	mv "${D}"/usr/lib/systemd/systemd-udevd "${D}"/sbin/udevd || die
-	mv "${D}"/usr/bin/udevadm "${D}"/bin/udevadm || die
-	dosym ../../../sbin/udevd /usr/lib/systemd/systemd-udevd
-	dosym ../../bin/udevadm /usr/bin/udevadm
-
 	# zsh completion
 	insinto /usr/share/zsh/site-functions
 	newins shell-completion/systemd-zsh-completion.zsh "_${PN}"
@@ -210,11 +193,14 @@
 
 	# Check whether we won't break user's system.
 	local x
-	for x in /bin/systemd /usr/bin/systemd \
-		/usr/bin/udevadm /usr/lib/systemd/systemd-udevd
+	for x in /bin/systemd /usr/bin/systemd
 	do
 		[[ -x ${D}${x} ]] || die "${x} symlink broken, aborting."
 	done
+
+	usr_wrap_bin systemadm udevadm
+	path-compat-mkwrap /sbin/udevadm /usr/bin/udevadm
+	path-compat-mkwrap /sbin/udevd /usr/lib/systemd/systemd-udevd
 }
 
 optfeature() {
